virttest/utils_test/libvirt.py

```diff
@@ -1592,6 +1592,22 @@ def alter_boot_order(vm_name, pci_id, boot_order=0):
     vmxml.sync()
 
 
+def wait_for_file_over(patt, xml_file, timeout=10):
+    """
+    Wait for file write over with pattern
+
+    :param patt: pattern for grep in file to wait, eg '</disk>'
+    :param xml_file: file for writing xml
+    :param timeout: wait_for timeout
+    """
+
+    def file_write_over():
+        if not process.run("grep %s %s" % (patt, xml_file),
+                           ignore_status=True).exit_status:
+            return True
+    utils_misc.wait_for(file_write_over, timeout)
```

Comment:
We'd better to log an error or raise exception here if there is no pattern in the file. 
for example, it could be :
```
if not utils_misc.wait_for(file_write_over, timeout):
    xxx
```

Reply:
If it can not find the pattern, just wait timeout, and maybe raise the same error as before like xxx file can not be find. Then what we should do is to increase the timeout. I don't want to deal with the return error here as it is meaningless and the purpose is just to wait. what do you think?

Reply:
Okay, got it.